src/Microsoft.Health.Fhir.Core/Features/Definition/SearchableSearchParameterDefinitionManager.cs

```diff
@@ -59,6 +72,10 @@ public SearchParameterInfo GetSearchParameter(string resourceType, string name)
             {
                 return parameter;
             }
+            else if (_fhirReqeustContextAccessor.FhirRequestContext.IncludePartiallyIndexedSearchParams && parameter.IsSupported)
```

Comment:
I think this and the one below should be 'or' cases for the original if statement sense they have the same body.